portal-ui/src/screens/Console/Tenants/HelpBox/TLSHelpBox.tsx

```diff
@@ -0,0 +1,124 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2022 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import React from "react";
+import { useSelector } from "react-redux";
+import { Box } from "@mui/material";
+import CertificateIcon from "../../../../icons/CertificateIcon";
+import { AppState } from "../../../../store";
+
+const FeatureItem = ({
+  icon,
+  description,
+}: {
+  icon: any;
+  description: string;
+}) => {
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        "& .min-icon": {
+          marginRight: "10px",
+          height: "23px",
+          width: "23px",
+          marginBottom: "10px",
+        },
+      }}
+    >
+      {icon}{" "}
+      <div style={{ fontSize: "14px", fontStyle: "italic", color: "#5E5E5E" }}>
+        {description}
+      </div>
+    </Box>
+  );
+};
+const TLSHelpBox = () => {
+  const namespace = useSelector((state: AppState) => {
+    return state.createTenant.fields.nameTenant.namespace || "<namespace>";
+  });
+
+  const tenantName = useSelector((state: AppState) => {
+    return state.createTenant.fields.nameTenant.tenantName || "<tenant-name>";
+  });
+
+  return (
+    <Box
+      sx={{
+        flex: 1,
+        border: "1px solid #eaeaea",
+        borderRadius: "2px",
+        display: "flex",
+        flexFlow: "column",
+        padding: "20px",
+        marginTop: {
+          xs: "0px",
+        },
+      }}
+    >
+      <Box
+        sx={{
+          display: "flex",
+          flexFlow: "column",
+        }}
+      >
+        <FeatureItem
+          icon={<CertificateIcon />}
+          description={`TLS Certificates Warning`}
+        />
+        <Box sx={{ fontSize: "14px", marginBottom: "15px" }}>
+          Automatic certificate generation is not enabled.
+          <br />
+          <br />
+          If you wish to continue only with <b>custom certificates</b> make sure
+          they are valid for the following internode hostnames , ie:
```

Comment:
Very minor formatting issue, should it be like below as i.e. seems to have dots [according to this](https://www.merriam-webster.com/words-at-play/ie-vs-eg-abbreviation-meaning-usage-difference):

```suggestion
          they are valid for the following internode hostnames, i.e.:
```